test/dev/encoding.test.js

```diff
@@ -22,7 +22,7 @@ describe('encoding', () => {
   })
 
   test('/ö/dynamic?q=food,coffee (encodeURIComponent)', async () => {
-    const { body: response } = await rp(url('/ö/dynamic?q=food%252Ccoffee'))
+    const { body: response } = await rp(url('/ö/dynamic?q=food,coffee'))
```

Comment:
`url` encodes this already - and I think we shouldn't be testing for double-decoding

Reply:
Context: #8429 / https://github.com/nuxt/nuxt.js/pull/8430

Reply:
Added tests to cover the issues reported in associated issues.